.travis.yml

```diff
@@ -1,22 +1,12 @@
 sudo: false
-
 language: php
-
 php:
-  - 5.4
   - 5.5
   - 5.6
   - 7.0
   - hhvm
-
-matrix:
-  allow_failures:
-    - php: hhvm
```

Comment:
I'm not so sure we want to keep up `hhvm` full support. It's nice to check compatibility but IMO we should allow it to fail.